R-packages/evalcast/R/manage_cards.R

```diff
@@ -91,3 +91,34 @@ intersect_averagers <- function(cards, grp_vars, avg_vars) {
   }
   cards %>% right_join(averagers_intersected)
 }
+
+#' Scale error measures based on those of a particular forecaster.
+#'
+#' @param score_card score_card like that returned by `evaluate_predictions()`
+#' @param vars vector of column names in `score_card` to normalize
+#' @param denom_forecaster name of forecaster in `score_card$forecaster` column by whose error
```

Comment:
Suggest `base_forecaster` or `baseline_forecaster` instead of `denom_forecaster`

Reply:
Done